packages/webapp/pages/_app.tsx

```diff
@@ -181,6 +182,7 @@ function InternalApp({ Component, pageProps, router }: AppProps): ReactElement {
         canonical={canonicalFromRouter(router)}
         titleTemplate={unreadCount ? `(${unreadText}) %s` : '%s'}
       />
+      {seo && <NextSeo {...seo} />}
```

Comment:
Why do we need NextSeo here if we already have DefaultSeo?
(not quite sure I get this one)

Reply:
I see you moved the Seo to layout :) 

Just a question does it overwrite ok now on pages like post and profile? (wasn't able to test the actual SEO output for Twitter/Slack on this domain)

Reply:
> Just a question does it overwrite ok

Yep. The default is just there for when NextSeo is not available. Basically they've co-existed since then, they are just side-by-side now.

> (wasn't able to test the actual SEO output for Twitter/Slack on this domain)

For post, yes, it does the SEO correctly. I will do the migration on all pages in a separate PR. And of course will do that and merge at the same time.

Reply:
Preview:

<img width="707" alt="Screenshot 2024-10-25 at 11 39 17 AM" src="https://github.com/user-attachments/assets/7930320d-5c83-4775-aa04-3ea451f9208c">